test/unify_package_rename_suggestors/unify_rename_suggestor_test.dart

```diff
@@ -0,0 +1,427 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react_codemod/src/unify_package_rename_suggestors/unify_rename_suggestor.dart';
+import 'package:test/test.dart';
+
+import '../resolved_file_context.dart';
+import '../util.dart';
+
+void main() {
+  final resolvedContext = SharedAnalysisContext.rmui;
+
+  // Warm up analysis in a setUpAll so that if getting the resolved AST times out
+  // (which is more common for the WSD context), it fails here instead of failing the first test.
+  setUpAll(resolvedContext.warmUpAnalysis);
+
+  group('UnifyRenameSuggestor', () {
```

Comment:
#nit It makes me a bit nervous that we're not testing the import renaming and the usage renaming together, to ensure they're in sync with what they update.

But, I don't think we have an easy way of testing codemod sequences, and I don't think it'd be worth trying to set that up.

Perhaps if they were updated within the same suggestor, then that'd enable easier testing. But again, I'm not sure refactoring it would be worth it.